src/js/components/ServiceSearchFilter.js

```diff
@@ -0,0 +1,65 @@
+import mixin from 'reactjs-mixin';
+/* eslint-disable no-unused-vars */
+import React from 'react';
+/* eslint-enable no-unused-vars */
```

Comment:
Btw – we could also just
```
// eslint-disable no-unused-vars
import React from 'react';
```
rather than having to disable/re-enable each time. cc @rcorral